modules/profile/manifests/jenkins/master.pp

```diff
@@ -270,4 +270,14 @@
     group => 'jenkins',
     content => template('jenkins_files/credentials.xml.erb'),
   }
+
+  cron {'docker image prune':
+    command => 'docker image prune -f'
```

Comment:
I believe this is missing a comma (`,`).

Reply:
Indeed I was. Thanks.